### ISSUE TITLE
Startup failure on MacOS Intel package

### ISSUE THREAD

**Issue by hubertp** [enso-org/enso#11076]
As reported internally when testing MacOS Intel(!) package.

```
(node:53927) UnhandledPromiseRejectionWarning: TypeError: Cannot read properties of undefined (reading '__wbindgen_add_to_stack_pointer')
    at parse3 (file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:70660:10)
    at parseEnso (file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:73426:16)
    at parseBlockWithSpans (file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:73826:16)
    at file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:77861:35
    at transact (file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:60902:14)
    at _Doc.transact (file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:59509:12)
    at _ModulePersistence.syncFileContents (file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:77845:19)
    at file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:77722:18
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async _ModulePersistence.withState (file:///Applications/Enso.app/Contents/Resources/app.asar/index.mjs:77683:20)
(Use `Enso --trace-warnings ...` to show where the warning was created)
(node:53927) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
```

**Comment by hubertp** [comment]
Maybe we could update `wasm-pack`, see for example https://github.com/rustwasm/wasm-bindgen/issues/3276.
We are on `0.12.1` but there is `0.13` available?

**Comment by hubertp** [comment]
One difference I see:
- MacOS GH runner uses `wasm-pack` 0.10.2
- MacOS hosted arm uses `wasm-pack` 0.12.1
And there have been issues reported again `wasm-pack` and `wasm-bindgen` against similar issues as the one reported here.

**Comment by hubertp** [comment]
Potentially fixed by #11078 and #11079.

**Comment by hubertp** [comment]
Confirmed as fixed. Closing.
